TrafficReplayer/src/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -193,6 +233,43 @@ public Stream<byte[]> getResponseDataStream() {
         }
     }
 
+    public static class RequestResponseResponseJSONTriple {
+        final JSONObject requestData;
+        final JSONObject primaryResponseData;
+        JSONObject shadowResponseData;
+
+        private JSONObject jsonFromHttpData(List<byte[]> data) throws IOException {
+            ByteArrayOutputStream requestStream = new ByteArrayOutputStream();
+            for(var d: data) {
+                requestStream.write(d);
+            }
+            String[] splitMessage = requestStream.toString(Charset.defaultCharset()).split("\r\n\r\n", 2);
+            JSONObject message = HTTP.toJSONObject(splitMessage[0].replaceAll("\r\n", "\n"));
```

Comment:
Why are you replacing \r\n with \n?  If this was a difference in behaviors between the two clusters, it would go unnoticed & could be rather fatal, especially if there's wonky code on the client side (presume that somebody will have a hand-built mobile client making a call).